src/Components/ServiceSelectionScene/ServiceSelectionScene.tsx

```diff
@@ -71,20 +85,49 @@ export class ServiceSelectionScene extends SceneObjectBase<ServiceSelectionScene
       body: new SceneCSSGridLayout({ children: [] }),
       $variables: new SceneVariableSet({
         variables: [
+          // Service search variable
           new CustomConstantVariable({
             name: VAR_SERVICE,
             label: 'Service',
             hide: VariableHide.hideVariable,
             value: '',
             skipUrlSync: true,
           }),
+          // Service name variable
+          new CustomConstantVariable({
+            name: SERVICE_NAME_VAR,
+            label: '',
+            hide: VariableHide.hideLabel,
+            value: SERVICE_NAME,
+            skipUrlSync: true,
+            options: [
+              {
+                value: SERVICE_NAME,
+                label: SERVICE_NAME,
+              },
+              {
+                value: AGGREGATED_SERVICE_NAME,
+                label: AGGREGATED_SERVICE_NAME,
+              },
+            ],
+          }),
         ],
       }),
-      $data: getQueryRunner([buildResourceQuery(`{${SERVICE_NAME}=~\`.*${VAR_SERVICE_EXPR}.*\`}`, 'volume')]),
+      $data: getSceneQueryRunner({
+        queries: [buildResourceQuery(`{${SERVICE_NAME_EXPR}=~\`.*${VAR_SERVICE_EXPR}.*\`}`, 'volume')],
+        runQueriesMode: 'manual',
+      }),
+      $labels: aggregatedMetricsEnabled
```

Comment:
what are we using labels for?

Reply:
To let us know if the `__aggregated_metrics__` label exists for that time range, just because the feature flag is enabled doesn't mean we have aggregated metrics for every service for every loki datasource? Or does it?